assets/base.css

```diff
@@ -2415,23 +2430,44 @@ input[type='checkbox'] {
   margin-right: -1.2rem;
 }
 
-@media screen and (max-width: 989px) {
-  menu-drawer ~ .header__icons .header__icon--account {
-    display: none;
-  }
-}
-
 .header__icon--menu[aria-expanded="true"]::before {
   content: "";
-  top: 100%;
+  bottom: 0;
   left: 0;
-  height: calc(var(--viewport-height, 100vh) - (var(--header-bottom-position, 100%)));
+  height: calc(var(--viewport-height, 100vh) - (var(--header-bottom-position, 100%)) - 1px);
   width: 100%;
   display: block;
-  position: absolute;
+  position: fixed;
   background: rgba(var(--color-foreground), 0.5);
 }
 
+@media screen and (max-width: 989px) {
+  menu-drawer ~ .header__icons .header__icon--account {
+    display: none;
+  }
+
+  .header__icon--menu[aria-expanded="true"]::before {
+    content: "";
+    top: 100%;
+    left: 0;
+    height: calc(var(--viewport-height, 100vh) - (var(--header-bottom-position, 100%)));
+    width: 100%;
+    display: block;
+    position: absolute;
+    background: rgba(var(--color-foreground), 0.5);
+  }
+}
+
+@media screen and (min-width: 990px) {
+  summary.header__icon--menu.drawer-open {
+    position: absolute;
+    left: 2rem;
```

Comment:
Hi @stufreen thanks for noticing that! The latest designs are on the Figma file which has the header not covered, there will be no change to the current drawer menu functionality on tablet.

Reply:
Thank you, I think between these sizes - it still works like desktop, where the X aligns with the nav links instead of the hamburger icon. I think that's acceptable, however we can keep the logo to stay on the left instead of jumping to the middle/center when the desktop logo layout is set to `middle left`. 

https://user-images.githubusercontent.com/88160784/211859297-1f25a412-d122-4539-8f51-f9d20f6aeb70.mov

Reply:
@eugenekasimov for context – @stufreen and I aligned on leaving this as is. No change needed for the padding for now.